kolibri/plugins/learn/assets/src/state/actions.js

```diff
@@ -259,17 +259,18 @@ function showLearnChannel(store, channelId, page = 1) {
         router.replace({ name: constants.PageNames.CONTENT_UNAVAILABLE });
         return;
       }
+      const isFacilityUser = coreGetters.isFacilityUser(store.state);
       const nextStepsPayload = { next_steps: session.user_id };
-      const popularPayload = { popular: session.user_id };
+      const popularPayload = { popular: 'true' };
```

Comment:
The bootloading of this data didn't work when it was a boolean here instead of a string (because in the Django template it was just interpreted as a string, not a Boolean, I guess -- and we pass it as a string as a GET param).